addons/sale/views/sale_portal_templates.xml

```diff
@@ -259,32 +253,28 @@
                                             <span class="fw-bold" t-field="sale_order.amount_total"/>
                                         </button>
                                     </div>
-                                    <p>
+                                    <div>
                                         <!-- The widget associated with this modal will hide and show divs in function of the amount selected. -->
-                                        <div id="o_sale_portal_use_amount_total">
-                                            By paying this proposal, I agree to the following terms:
-                                        </div>
-                                        <div t-if="prepayment_available"
-                                             id="o_sale_portal_use_amount_prepayment"
-                                        >
-                                            By paying this <u>down payment</u> of
-                                            <span t-esc="prepayment_amount" class="fw-bold"
-                                                  t-options="{'widget': 'monetary', 'display_currency': sale_order.currency_id}"/>
-                                            (<b t-esc="sale_order.prepayment_percent * 100"/> %)
-                                            for this proposal, I agree to the following terms:
-                                        </div>
-                                        <ul>
-                                            <li>
-                                                <span>Accepted on the behalf of:</span> <b t-field="sale_order.partner_id.commercial_partner_id"/>
-                                            </li>
-                                            <li>
-                                                <span>For an amount of:</span> <b data-id="total_amount" t-field="sale_order.amount_total"/>
-                                            </li>
-                                            <li t-if="sale_order.payment_term_id">
-                                                <span>With payment terms:</span> <b t-field="sale_order.payment_term_id.note"/>
-                                            </li>
-                                        </ul>
-                                    </p>
+                                        <span t-if="prepayment_available">
+                                            <span id="o_sale_portal_use_amount_prepayment"> 
+                                                By paying a <u>down payment</u> of 
+                                                <span t-esc="prepayment_amount" class="fw-bold"
+                                                      t-options="{'widget': 'monetary', 'display_currency': sale_order.currency_id}"/>
+                                                (<b t-esc="sale_order.prepayment_percent * 100"/>%)
```

Comment:
Missing comma and dot at the end.
![image](https://github.com/odoo/odoo/assets/15896813/9d1039cc-3ee5-4da5-9631-db917fc3c9ba)